controllers/utils.go

```diff
@@ -232,7 +232,7 @@ func validateRolloutsScope(cr rolloutsmanagerv1alpha1.RolloutManager, namespaceS
 			return &reconcileStatusResult{
 				rolloutController: &phaseFailure,
 				phase:             &phaseFailure,
-			}, fmt.Errorf(UnsupportedRolloutManagerNamespaceScoped)
+			}, errors.New(UnsupportedRolloutManagerNamespaceScoped)
```

Comment:
The changes in this file are already present in the main branch 🤔  Can you try to rebase again?

Reply:
Yes, When I raised this PR, it was not there, rebased it now